server/core/region_tree.go

```diff
@@ -28,8 +28,8 @@ type regionItem struct {
 
 // Less returns true if the region start key is less than the other.
 func (r *regionItem) Less(other btree.Item) bool {
-	left := r.region.GetStartKey()
-	right := other.(*regionItem).region.GetStartKey()
+	left := r.region.StartKey
+	right := other.(*regionItem).region.StartKey
```

Comment:
Will this modification bring performance gains?

Reply:
I will bench it and send another PR.